pygrametl/tables.py

```diff
@@ -1898,6 +1898,11 @@ def __insertexecutemany(self):
             self.targetconnection.executemany(self.insertsql, self.__batch)
             self.__batch = []
 
+    @property
+    def currentsize(self):
```

Comment:
Could `currentsize` be misunderstood as the size of the table in the database? Maybe `currentbatchsize` could be used, but then the API differs between batch loading and bulk loading Dimensions/FactTables.

Reply:
Yes, I think we should use the same name in both places. I also thought about "awaiting". Is that easier to understand?